Weaning.Rmd

```diff
@@ -0,0 +1,39 @@
+Citation: Trivers R.L. 1974. Parent-Offspring Conflict.  Amer. Zool. 14:249-264.
+
+##Selfish Pups: Weaning Conflict and Milk Theft in Free-Ranging Dogs
+-	This paper focuses on weaning conflict in Canis familiaris
+-	Suckling behaviour was observed in pups since birth, and the weaning period was estimated to happen at 7th through 13th week of age, i.e. mothers refused to nurse, when the pups reached the age of 7 weeks
+-	They ran a GLMM model, which showed that there exists a three-way interaction between idenitity of suckling initiator, pup age and litter size
+-	As pups grow in age, the amount of suckling initiations by pups increase, and initiations by mother decrease
+-	Pups have been shown to be involved in milk theft, i.e. nursing from unrelated lactating females 
+Citation: Manabi P. and Bhadra A. Selfish Pups: Weaning Conflict and Milk Theft in Free-Ranging Dogs.  PLOS One, 12(2): e0170590.
+
+
+
+##Maternal care in house mice (Mus muscukus): II. The energy cost of lactation as a function of litter size
+-	This paper  evaluate lactation performance in house mice,  in relation to litter size
+-	For larger litter size, the female increased both the total milk production, and the amount of nutrients present in milk 
+-	Mortality was higher for larger litter size, possibly the females adjust the litter size in order to optimize the ratio of milk production cost to the litter size 
+Citation: Kounig B., Riester J., Markl H. Maternal care in house mice (Mus muscukus): II. The energy cost of lactation as a function of litter size.  Journal of Zoology 216(2): 195-210
+
+##Lactation, maternal behavior and infant growthin common marmoset monkeys (Callithrix jacchus): effects of maternal size and litter size
```

Comment:
So marmosets are actually super cool! I highly recommend reading more about them for fun =) Here are some highlights from my previous readings of [wikipedia](https://en.wikipedia.org/wiki/Marmoset) and [NY Times](http://www.nytimes.com/2007/03/27/science/27marm.html) (em added):

> 95% of Marmoset fraternal twins trade blood through chorionic fusions, making them hematopoietic chimeras.[7][8]  

> It has recently been discovered that marmosets can carry the reproductive cells of their (fraternal) twin siblings, because of placental fusion during development. (Marmosets almost always give birth to fraternal twins.)[5][11][12]

> In other words, the sperm came from one male, but it had the DNA of the male’s brother. A paternity test would show that the baby’s genetic father was actually its uncle. The scientists were not able to isolate DNA from marmoset eggs, but they did find that 2 out of 21 marmoset ovaries were chimeric. It’s possible that a female marmoset can give birth to nephews and nieces.

> That common bond might drive the evolution of unusual kinds of behavior. Dr. Haig notes that in a band of marmosets, only the oldest female can reproduce, while the younger individuals help raise her offspring.

> “This changes how we think of marmosets as individuals, but it also changes how we think of the term at all,” she said. **A male mates with a female, who gives birth to his brother’s offspring. “But most of his body also has his brother’s genes. So what is he as an individual?”**

Reply:
That's interesting! 
I'm actually working on a project trying to understand sexual selection at the gametic level in marmosets for my other class. So I agree, marmosets ARE cool!